Frameworks/CoreText/CTRun.mm

```diff
@@ -106,8 +106,7 @@ void CTRunGetGlyphs(CTRunRef run, CFRange range, CGGlyph buffer[]) {
 
 /**
  @Status Caveat
- @Notes Only font face, size and text color attributes are supported.  Background fill is
-        always white.
+ @Notes Only the starting position of the glyph run is handled.
```

Comment:
With DWrite we do not have access to _glyphOrigins. I tried calculating it myself with whatever DWrite provided back to me but it was way off.

---
In reply to: [77379732](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77379732) [](ancestors = 77379732)